pkg/commands/branches.go

```diff
@@ -101,6 +101,39 @@ func (c *GitCommand) GetBranchUpstreamDifferenceCount(branchName string) (string
 	return c.GetCommitDifferences(branchName, branchName+"@{u}")
 }
 
+// TryGetMainBranch tries to get the main branch
+// For old repo's this is usually master and for newer once it is main
+func (c *GitCommand) TryGetMainBranch() string {
+	// Firstly try to get the main branch from the origins
+	cmdStr := `git branch --remotes --list '*/HEAD' --format '%(symref:short)'`
+	output, _ := c.OSCommand.RunCommandWithOutput(cmdStr)
+	lines := strings.Split(strings.TrimSpace(output), "\n")
+	for _, line := range lines {
+		parts := strings.SplitN(line, "/", 2)
+		if len(parts) == 2 {
+			return parts[len(parts)-1]
+		}
+	}
+
+	// If we didn't find a main branch from the remotes we check if there is a main or master branch
+	cmdStr = `git branch --format "%(refname:short)" --list "master" --list "main"`
+	output, _ = c.OSCommand.RunCommandWithOutput(cmdStr)
+	lines = append(lines[:0], strings.Split(strings.TrimSpace(output), "\n")...)
+	for _, line := range lines {
+		if line == "master" || line == "main" {
+			return line
+		}
+	}
+
+	return ""
+}
+
+func (c *GitCommand) HasDevelopmentBranch() bool {
+	cmdStr := `git branch --format "%(refname:short)" --list "development"`
+	output, _ := c.OSCommand.RunCommandWithOutput(cmdStr)
+	return strings.TrimSpace(output) == "development"
```

Comment:
this should be 'develop' not 'development' I believe

Reply:
I've also seen repositories use "dev".

Reply:
Dunno i've always used `development` as development branch

Reply:
Interesting. Git flow uses the `feature/foo` naming and it uses `develop` too so I think we should leave that as the value here, but I'm open to have this configurable against the repo in the lazygit config

Reply:
Why not support all three of them? I can see no harm

Reply:
I see two potential issues that could arise from supporting all of them:
1) if a random branch from ages ago exists called 'dev' we might treat that as the 'develop' branch and it would put things out of whack. Same with vice-versa if you actually use 'dev'. I know my work branches are cluttered with heaps of random crap from people making quick PRs
2) it may be expensive at runtime to check against each possible develop branch

Reply:
Changed it to `develop`